src/Reflector.php

```diff
@@ -87,18 +87,27 @@ final class Reflector
     /**
      * @param array<string,string>|null $symbols
      * @param array<string,string>      $source
-     * @param string[]                  $missingSymbols
+     * @param array<string, int>        $missingSymbols
      */
     private static function initSymbolList(?array &$symbols, array $source, array $missingSymbols): void
     {
         if (null !== $symbols) {
             return;
         }
 
+        $excludingSymbols = array_keys(
+            array_filter($missingSymbols, static function ($version) {
+                return PHP_VERSION_ID < $version;
+            })
+        );
+
         $symbols = array_fill_keys(
-            array_merge(
-                array_keys($source),
-                $missingSymbols
+            array_diff(
```

Comment:
You should add `use function array_diff` on top of the file.

Reply:
fixed